pkg/monitor/cluster/apiservercount.go

```diff
@@ -0,0 +1,34 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitAPIServerCount(ctx context.Context) error {
+	ds, err := mon.cli.AppsV1().DaemonSets("openshift-apiserver").Get("apiserver", metav1.GetOptions{})
+	if err != nil {
```

Comment:
Maybe something like this? 
```suggestion
	if err != nil {
       	mon.emitGauge("apiserver.openshift.count", 0)
```

Reply:
That makes sense to me

Reply:
This is kind of misleading, isn't it? If we failed to get this data, it should be unknown (i.e. not emitted), not 0 IMO.